### PR TITLE
Add nature2021 dataset.

### PR DESCRIPTION
**Pull request by eltee1**

### CODE REVIEW

File: source/database/src/data/sql/datasets/load.sql [hunk 1]
```diff
@@ -26,3 +26,18 @@ INSERT INTO i18n_datasets (code, language_code, name, description)
...
 {import_common 'nature2020/emission_factors'}
+
+
+-- Dataset(s) voor AERIUS 2021
+
+INSERT INTO datasets (code, name, schema_name, current, description)
+	VALUES ('nature2021', 'AERIUS 2021 nature', 'nature2021', true, 'Dataset used in AERIUS 2021, nature profile, to be released...');
+
+INSERT INTO i18n_datasets (code, language_code, name, description)
+	VALUES ('nature2021', 'nl', 'AERIUS 2021 natuur', 'Dataset zoals gebruikt in AERIUS 2021, natuur profiel, nog niet publiek beschikbaa');
```
Comment (BertScholten) [inline source/database/src/data/sql/datasets/load.sql:37]: beschikbaa**r**?

File: source/database/src/data/sql/datasets/load.sql [hunk 2]
```diff
@@ -26,3 +26,18 @@ INSERT INTO i18n_datasets (code, language_code, name, description)
...
+INSERT INTO i18n_datasets (code, language_code, name, description)
+	VALUES ('nature2021', 'en', 'AERIUS 2021 nature', 'Dataset used in AERIUS 2021, nature profile, to be released...');
+
+{import_common 'nature2021/dataset_metadata.sql'}
+{import_common 'nature2021/sectors'}
+{import_common 'nature2021/emission_factors'}
```
Comment (BertScholten) [inline source/database/src/data/sql/datasets/load.sql:43]: Please include a newline at the end of each file.
(not having one can affect future comparisons, which makes it slightly more annoying to review)

File: source/database/src/data/sql/nature2021/dataset_metadata.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO nature2021.dataset_metadata (key, value) VALUES ('MINIMUM_YEAR', '2018');
+INSERT INTO nature2021.dataset_metadata (key, value) VALUES ('MAXIMUM_YEAR', '2030');
```
Comment (BertScholten) [inline source/database/src/data/sql/nature2021/dataset_metadata.sql:2]: Wasn't the maximum 2040 for nature2021?

File: source/database/src/data/sql/nature2021/sectors/load.sql [hunk 4]
```diff
@@ -0,0 +1,8 @@
+BEGIN; SELECT setup.ae_load_table('nature2020.sectors', '{data_folder}/temp/temp_sectors_20200527.txt'); COMMIT;
```
Comment (BertScholten) [inline source/database/src/data/sql/nature2021/sectors/load.sql:1]: Wrong schema used in this file

File: source/database/src/data/sql/nature2021/sectors/load_i18n.sql [hunk 5]
```diff
@@ -0,0 +1,6 @@
+-- i18n voor de module sectors
+BEGIN; SELECT setup.ae_load_table('nature2020.i18n_sectors', '{data_folder}/temp/i18n.temp_sectors_20200527_nl.txt'); COMMIT;
```
Comment (BertScholten) [inline source/database/src/data/sql/nature2021/sectors/load_i18n.sql:2]: Wrong schema used in this file

File: source/database/src/main/sql/load/02-main_datasets.sql [hunk 6]
```diff
@@ -1,2 +1,2 @@
 CREATE SCHEMA nature2020;
-{import_common_into_schema 'template/', 'nature2020'}
+{import_common_into_schema 'template/', 'nature2020'}
```
Comment (BertScholten) [inline source/database/src/main/sql/load/02-main_datasets.sql:2]: No newline at end of file

File: source/database/src/main/sql/load/03-nature2021_dataset.sql [hunk 7]
```diff
@@ -0,0 +1,2 @@
+CREATE SCHEMA nature2021;
+{import_common_into_schema 'template/', 'nature2021'}
```
Comment (BertScholten) [inline source/database/src/main/sql/load/03-nature2021_dataset.sql:2]: No newline at end of file

**Review by BertScholten — CHANGES_REQUESTED** [review]
I know it's a work-in-progress, but spotted some points when glancing at it.

File: source/database/src/data/sql/nature2021/dataset_metadata.sql (hunk 3 above)
Comment (eltee1) [reply source/database/src/data/sql/nature2021/dataset_metadata.sql:2]: I heard it would be 2035, also heard is would be 2040, so left it as it is...

**Review by BertScholten — APPROVED** [review]
LGTM

**Pull request merged**